portfolio/src/main/webapp/script.js

```diff
@@ -49,9 +49,39 @@ function getComment() {
 
 }
 
+function getUserInfo(){
+  fetch('/loggedin').then(response => response.json()).then((userInfo) => {
+    const statsListElement = document.getElementById('comment-section-container');
+    statsListElement.innerHtML = '';
+    if(userInfo.isUserLoggedIn){
+      const liElement = document.createElement('li');
+      liElement.innerHTML = '<h2>Comments</h2>'+
+      '<form action="/data" method = "POST">'+
+        '<p>Logged in as "'+ userInfo.email +'"</p>'+
+        '<p>Choose number of comments</p>'+
+        '<input type="number" name="max-number" min="0" max="10" ><br>'+
+        '<textarea placeholder="Enter a message" name="text-input" cols="30" rows="3"></textarea>'+
+        '<input type="submit" />'+
+        '<div id="comment-container"></div>'+
+        '<p>Logout <a href="'+ userInfo.logInUrl +'">here</a>.</p>'+
+      '</form>';
```

Comment:
this is a lot for building from a string. I think it is going to be hard to change or maintain. Maybe instead it would be better to build this using document.createElement and element.appendChild()?

Another option is to declare the elements in html, and then hide them with js.

I think that on big projects, you usually use a library to help with this kind of stuff. Maybe brooke has an opinion on how to make this easier.